--- a/libs/parser/lkc.h	2024-09-21 18:32:55.939528260 +0200
+++ b/libs/parser/lkc.h	2024-09-22 08:52:16.911917804 +0200
@@ -30,6 +30,7 @@
 #undef CONFIG_
 #define CONFIG_ CONFIG_prefix()
 
+#define yylineno zconflineno
 extern int yylineno;
 void zconfdump(FILE *out);
 void zconf_starthelp(void);
@@ -61,6 +62,7 @@
 char *xstrndup(const char *s, size_t n);
 
 /* lexer.l */
+#define yylex zconflex
 int yylex(void);
 
 struct gstr {
@@ -102,7 +104,7 @@
 struct menu *menu_get_parent_menu(struct menu *menu);
 bool menu_has_help(struct menu *menu);
 const char *menu_get_help(struct menu *menu);
-struct gstr get_relations_str(struct symbol **sym_arr, struct list_head *head);
+void get_relations_str(struct symbol **sym_arr, struct list_head *head, struct gstr *res);
 void menu_get_ext_help(struct menu *menu, struct gstr *help);
 
 /* symbol.c */
