--- a/libs/parser/lkc_proto.h	2024-09-22 09:47:27.077925132 +0200
+++ b/libs/parser/lkc_proto.h	2024-09-24 20:21:29.017466167 +0200
@@ -23,7 +23,7 @@
 struct menu * menu_get_parent_menu(struct menu *menu);
 bool menu_has_help(struct menu *menu);
 const char * menu_get_help(struct menu *menu);
-struct gstr get_relations_str(struct symbol **sym_arr, struct list_head *head);
+void get_relations_str(struct symbol **sym_arr, struct list_head *head, struct gstr *res);
 void menu_get_ext_help(struct menu *menu, struct gstr *help);
 
 /* symbol.c */
