--- a/libs/parser/yconf.y	2012-01-12 22:37:17.582339777 +0100
+++ b/libs/parser/yconf.y	2012-03-03 23:55:29.889737630 +0100
@@ -127,7 +127,8 @@
 	 * later regardless of whether it comes from the 'prompt' in
 	 * mainmenu_stmt or here
 	 */
-	menu_add_prompt(P_MENU, xstrdup("Linux Kernel Configuration"), NULL);
+	menu_add_prompt(P_MENU, xstrdup(ROOTMENU), NULL);
+
 };
 
 
@@ -746,7 +746,7 @@
 	}
 }
 
-#include "zconf.lex.c"
+#include "lconf.c"
 #include "util.c"
 #include "confdata.c"
 #include "expr.c"
