--- a/frontends/nconf/nconf.c	2024-09-22 09:47:27.077925132 +0200
+++ b/frontends/nconf/nconf.c	2024-09-24 20:06:50.334603758 +0200
@@ -724,7 +724,8 @@
 		dialog_input += strlen(CONFIG_);
 
 	sym_arr = sym_re_search(dialog_input);
-	res = get_relations_str(sym_arr, NULL);
+	res = str_new();
+	get_relations_str(sym_arr, NULL, &res);
 	free(sym_arr);
 	show_scroll_win(main_window,
 			"Search Results", str_get(&res));
