diff --git a/frontends/gconf/gconf.c b/frontends/gconf/gconf.c
--- a/frontends/gconf/gconf.c
+++ b/frontends/gconf/gconf.c
@@ -1444,8 +1444,10 @@
 int main(int ac, char *av[])
 {
 	const char *name;
+#if 0
 	char *env;
-	gchar *glade_file;
+#endif
+	gchar *glade_file = GUI_PATH;
 
 	/* GTK stuffs */
 	gtk_set_locale();
@@ -1455,6 +1455,7 @@
 	//add_pixmap_directory (PACKAGE_DATA_DIR "/" PACKAGE "/pixmaps");
 	//add_pixmap_directory (PACKAGE_SOURCE_DIR "/pixmaps");
 
+#if 0
 	/* Determine GUI path */
 	env = getenv(SRCTREE);
 	if (env)
@@ -1463,6 +1464,7 @@
 		glade_file = g_strconcat(av[0], ".glade", NULL);
 	else
 		glade_file = g_strconcat(g_get_current_dir(), "/", av[0], ".glade", NULL);
+#endif
 
 	/* Conf stuffs */
 	if (ac > 1 && av[1][0] == '-') {
